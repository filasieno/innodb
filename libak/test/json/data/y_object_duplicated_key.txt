name=y_object_duplicated_key
max_depth=32
max_string_size=2048
max_json_size=1048576
----------
{"a":"b","a":"c"}