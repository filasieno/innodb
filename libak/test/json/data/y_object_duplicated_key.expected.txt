---
STATE_CHANGED_EVENT: STATE_INITIALIZED
BEGIN_OBJECT
ATTR_KEY "a" more=0
STRING_VALUE "b" more=0
ATTR_KEY "a" more=0
STRING_VALUE "c" more=0
END_OBJECT
STATE_CHANGED_EVENT: STATE_DONE
